common/autoware_ad_api_msgs/routing/srv/SetRoutePoints.srv

```diff
@@ -0,0 +1,9 @@
+std_msgs/Header header
+geometry_msgs/Pose[<=1] start
+geometry_msgs/Pose goal
+geometry_msgs/Pose[] waypoints
```

Comment:
By the way, is `via_points` better?
@LalithVipulananthan @maxime-clem What do you think? They are the points where the vehicle must drop by before arriving at the goal.

![image](https://user-images.githubusercontent.com/31987104/183536246-0bd7fcc2-d3b1-42be-a81b-32c36386ea14.png)

```suggestion
geometry_msgs/Pose[] via_points
```

Reply:
Way point is a more common word. I do not think this change is needed.

Reply:
Okay, thanks! I'll resolve this.